Applications/Piano/PianoWidget.h

```diff
@@ -55,4 +59,15 @@ class PianoWidget final : public GWidget {
     bool m_release_enabled { false };
 
     bool keys[256];
+
+#define key_count sizeof(piano_keys) / sizeof(PianoKey)
+
+    struct KeyRect {
+        PianoKey key;
+        Rect rect;
+    };
+    KeyRect m_key_rects[key_count] = {};
+
+    int m_mouse_over { -1 };
```

Comment:
Let's come up with a more descriptive name for this one.. `m_piano_key_under_mouse` perhaps?

Reply:
Works for me!